Framework/Algorithms/src/SampleCorrections/MCInteractionVolume.cpp

```diff
@@ -72,6 +70,26 @@ const Geometry::BoundingBox &MCInteractionVolume::getBoundingBox() const {
   return m_sample->getBoundingBox();
 }
 
+/**
+ * Returns the axis-aligned bounding box for the volume including env
+ * @return A reference to the bounding box
+ */
+const Geometry::BoundingBox MCInteractionVolume::getFullBoundingBox() const {
+  auto sampleBox = m_sample->getBoundingBox();
+  if (m_env) {
+    try {
+      const auto &envBox = m_env->boundingBox();
+      sampleBox.grow(envBox);
+    } catch (std::runtime_error &) {
+    }
```

Comment:
Should this catch be doing anything with the runtime error?

Reply:
I'll remove it. I moved this code from RectangularBeamProfile.cpp where it had some value but in the new unit the code it surrounds won't ever raise an exception.